notebooks/GSLIBComparison.jl

```diff
@@ -0,0 +1,290 @@
+### A Pluto.jl notebook ###
+# v0.12.11
+
+using Markdown
+using InteractiveUtils
+
+# ╔═╡ 20c69a8e-1fa2-11eb-3f1e-ef154de99450
+begin
+  using Distributed
+  pids = [myid()]
+
+  md"""
+  Running on processes: $pids
+
+  Use `pids = addprocs(n)` to run the notebook with `n` parallel processes.
+  """
+end
+
+# ╔═╡ 2399f800-1fa2-11eb-185d-53d05516dacf
+@everywhere pids begin
+  using Pkg; Pkg.activate(@__DIR__)
+  Pkg.instantiate(); Pkg.precompile()
+end
+
+# ╔═╡ 2c2e7df6-1fa2-11eb-29a8-3b59d382267e
+@everywhere pids begin
+  # packages used in this notebook
+  using GeoStats
+  using FileIO
+  using GslibIO
+
+  # default plot settings
+  using Plots; gr(size=(700,400))
+end
+
+
+# ╔═╡ 524c65d0-2ea9-11eb-0450-5ff97535406a
+md"""
+# GSLIB comparison
+
+This tutorial shows how to run a Kriging estimation problem that homologates GSLIB kriging program *kt3d*.
+
```

Comment:
Wouldn't it be an homologation of GeoStats.jl using kt3d as reference?

Reply:
I agree @rmcaixeta , I think the English word `homologation` applies to GeoStats.jl with GSLIB as a reference. The sentence written above seems to state the opposite direction.

Reply:
Maybe it is better to say "reproduce" instead "homologate". Or perhaps a comparison only.

Reply:
Reproduce is a simpler and better term :100: